src/apps/workspaces/domain/workspace.py

```diff
@@ -199,6 +216,8 @@ class PublicWorkspaceManager(PublicModel):
     is_pinned: bool = False
     applets: list[WorkspaceManagerApplet] | None = None
     title: str | None
```

Comment:
Is this still being returned by the query?

Reply:
yes

Reply:
Confirmed in Swagger UI there's no issues with `title` being returned:

<img width="262" alt="Screenshot 2024-06-13 at 10 01 42 AM" src="https://github.com/ChildMindInstitute/mindlogger-backend-refactor/assets/1282772/20975d25-8f97-4490-a559-c01833047574">